deploy/chart/templates/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -17,6 +17,13 @@ spec:
       labels:
         app: olm-operator
     spec:
+      securityContext:
+        runAsNonRoot: true
+        seccompProfile:
+          type: RuntimeDefault
+        {{- if eq .Values.installType "upstream" }}
+        runAsUser: {{ .Values.package.securityContext.runAsUser }}
```

Comment:
@anik120 @perdasilva,
 
In "downstream" if we omit this value when the image (Dockerfile) does not set the USER ID that will fail with the error `container has runAsNonRoot and image has non-numeric user`.

Please, test this scenario with an image that requires the RunAsUserID example: 'memcached:1.4.36-alpine'
Example : https://gist.github.com/camilamacedo86/c7c06e51272f8c348655123d0f18eceb